CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
+## [1.2.9] - 2023-09-20
+### Breaking Changes
```

Comment:
Also add information on whether are recommending users to upgrade the SDK or this is just information/suggestion.

Reply:
okay